exporter/awsxrayexporter/internal/translator/segment_test.go

```diff
@@ -514,7 +505,7 @@ func TestSpanWithAnnotationsAttribute(t *testing.T) {
 
 	assert.NotNil(t, segment)
 	assert.Equal(t, 1, len(segment.Annotations))
-	assert.Equal(t, "val2", segment.Annotations["attr2_2"])
+	assert.Equal(t, "val2", segment.Annotations["attr2@2"])
```

Comment:
Can we confirm with X-Ray backend team that what will be the consequence if customers try to index attributes with some other special chars such as @, #, *, etc?

Reply:
Yeah sorry I made a mistake, it's add . support only :( let me fix it

Reply:
Get a clarify here: Otel should only send "." type input, so we don't need to handle and test anything else here

Reply:
https://opentelemetry.io/docs/specs/semconv/general/attribute-naming/

Reply:
OTEL follows that naming convention, but customers can provide their own attributes and don't have to follow it.

Reply:
the backend (and us) should only limit our chart to Otel support, if customer decided to send their own naming attributes, we should ignore it.

Reply:
OK so after discuss with our PM, we decided to keep the converter, but just pass-through "." as is